### ISSUE TITLE
planet init fails

### ISSUE THREAD

**Issue by strixcuriosus** [planetlabs/planet-client-python#148]
`planet init` seems to be broken:

```
planet init
Email: asdf@example.com
Password:
Error: Unexpected response: invalid parameters, login process has changed
```

planet 1.1.0
Python 2.7.15

**Comment by sarasafavi** [comment]
Thanks for the report @strixcuriosus ! Fix is in the works.

**Comment by craigdsouza** [comment]
Hello.. has there been any resolution here? I'm trying to run pclip commands and I'm getting the same error.. thanks..
Python version is 2.7.15, Windows 10, pclip 0.2.7

**Comment by sarasafavi** [comment]
@craigdsouza @strixcuriosus this issue is now fixed in the latest release, 1.2.1: https://pypi.org/project/planet/1.2.1/
